README.md

```diff
@@ -240,25 +243,28 @@ Some handy links:
 The web API exposes this schema in two ways:
 
 - Swagger UI: `http://localhost:8000/docs`
-    - You can use this to construct requests to the server
-    - Make sure you fill in the Bearer token (at the top right )
+  - You can use this to construct requests to the server
+  - Make sure you fill in the Bearer token (at the top right )
 - OpenAPI schema: `http://localhost:8000/schema.json`
-    - Returns a JSON with the full OpenAPI 3 compliant schema.
-    - You could put this into the [Swagger editor](https://editor.swagger.io/) to see the same "Swagger UI" that `/api/docs` exposes.
-    - We generate the sample_metadata installable Python API based on this schema.
+  - Returns a JSON with the full OpenAPI 3 compliant schema.
+  - You could put this into the [Swagger editor](https://editor.swagger.io/) to see the same "Swagger UI" that `/api/docs` exposes.
+  - We generate the sample_metadata installable Python API based on this schema.
 
 #### Generating the installable API
 
 The installable API is automatically generated through the `package.yml` GitHub action and uploaded to PyPI.
 
-
 To generate the python api you'll need to install openapi generator v5.x.x
 
+Follow [this tutorial](https://openapi-generator.tech/docs/installation/) for
```

Comment:
Can you list the latest 5.x.x release here, and move instructions to download that release to here?

Reply:
Sure thing